.idea/libraries/org_antlr_antlr4_4_5_1_1.xml

```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
```

Comment:
No problem, I was just thinking out loud :-)
Can you remove the .idea directory though?